docs/latest/releaseNotes.yml

```diff
@@ -39,6 +39,81 @@ docDescription: >-
 changelog: https://github.com/telepresenceio/telepresence/blob/$branch$/CHANGELOG.md
```

Comment:
are you sure we need to update this changelog ? I though it was redirected to `getambassador.io`

Reply:
we probably dont need to 🤷